glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/MetricsPingScheduler.kt

```diff
@@ -31,8 +34,11 @@ import java.util.concurrent.TimeUnit as AndroidTimeUnit
  * - ping is overdue (due time already passed) for the current calendar day;
  * - ping is soon to be sent in the current calendar day;
  * - ping was already sent, and must be scheduled for the next calendar day.
+ *
+ * The scheduler also makes use of the [LifecycleObserver] in order to correctly schedule
```

Comment:
Do these btw get correctly linked?

Reply:
They should, I've not checked in the glean-core repo but this does work in the docs that are generated in A-C